lk.toml

```diff
@@ -0,0 +1,4 @@
+# Example workspace configuration toml file
+
+# scripts_dir = "./lk/tests/depends_on_file/"
+# default_mode = "list"
```

Comment:
This file isn't used and isn't clearly marked as an example and I think perhaps it should be one or the other. 

Going to use this comment to bring together a few related thoughts. I think this PR is super-great and I'm really grateful for it. Love that `config.rs` doesn't need to exist any more.

There are two broad issue:
1. Naming: this is essentially an "include" feature, but without an "exclude". It needs an exclude because of `executables.rs#L33`!
2. Updating the config file manually.

### Naming
While I don't mind `scripts_dir` I suspect `include` and `exclude` might be more idiomatic ways of framing this. E.g. someone might want to keep their scripts exactly where they are but exclude other dirs, e.g. `node_modules`. Of course `node_modules` is already manually excluded in `executables.rs#L33`, but I don't like that list! In fact I really want it gone because this should all already be in config. If it were, I might call it `exclude`. This means a companion `include` makes quite a bit of sense. 

If we specify a local exclude we probably expect `lk` to still use the excludes in the global config. Having separate configs really starts to raise these kind of questions. Do you know if this is supported in the library you added?

### Updating the config

#### Manually
If I wanted to specify a local dir for scripts I'd need to manually create a local `lk.toml` config file, or add it to the global config. This isn't a super-great experience. I'd rather it be possible to set this on the command line, like I can currently do with default, e.g. `lk --default fuzzy` (you've removed this because the logic for it was in `config.rs`, which rightly had to go.

#### Using args
If we set config values via `lk` (and I think that's best) then we need to tell `lk` where to set the value, either locally or globally. Options:
1. have two of each, e.g. `lk --include ...` and `lk --include-global ...`.
4. `lk --default-global fuzzy` it could be `lk --default fuzzy --global` or the reverse, `lk --default fuzzy --local`.

I think we can do without args, but need an easy way to edit config. Some ideas:
1. `lk --edit-[local|global]]-config` opens the config in the default editor
2. `lk --help` shows the location of the configs
3. `lk --create-local-config` adds a template local config with examples.

What are your thoughts on this?

Reply:
Maybe there's too much here and it could be a subsequent PR. If you'd rather that then just changing the `script-dir` to `include` and making it a list would be enough.

Reply:
Oh, and adding the location of the config file to the help.